hack/common.sh

```diff
@@ -44,9 +44,9 @@ NOOBAA_CSV="$OUTDIR_TEMPLATES/noobaa-csv.yaml"
 ROOK_CSV="$OUTDIR_TEMPLATES/rook-csv.yaml.in"
 OCS_CSV="$OUTDIR_TEMPLATES/ocs-operator.csv.yaml.in"
 
-LATEST_ROOK_IMAGE="rook/ceph:v1.8.3"
-LATEST_NOOBAA_IMAGE="noobaa/noobaa-operator:master-20210912"
-LATEST_NOOBAA_CORE_IMAGE="noobaa/noobaa-core:master-20210912"
+LATEST_ROOK_IMAGE="rook/ceph:v1.9.9"
```

Comment:
We really don't expect to use a 1.9.9 image in the 4.10 branch, which is based on 1.8

Reply:
reverted, we should be 1.8 based only now